### PR TITLE
FEATURE: pkg/exchange: add query market to bybit exchange

### PR DESCRIPTION
**Pull request by bailantaotao**

### CODE REVIEW

File: pkg/exchange/bybit/convert.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	//    MinOrderQty: 0.000048
+	//    MaxOrderQty: 71.73956243
+	//    MinOrderAmt: 1
+	//    MaxOrderAmt: 2000000
+	//}
+	//
+	//PriceFilter:
+	//{
+	//    TickSize: 0.01
+	//}
```
Comment (gx578007) [inline pkg/exchange/bybit/convert.go:32]: Could you add a unit test for this converting function to make sure parsing is performed as expected?

**Review by c9s — APPROVED** [review]

File: pkg/exchange/bybit/convert.go [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+	return types.Market{
+		Symbol:          m.Symbol,
+		LocalSymbol:     m.Symbol,
+		PricePrecision:  int(math.Log10(m.LotSizeFilter.QuotePrecision.Float64())),
+		VolumePrecision: int(math.Log10(m.LotSizeFilter.BasePrecision.Float64())),
+		QuoteCurrency:   m.QuoteCoin,
+		BaseCurrency:    m.BaseCoin,
+		MinNotional:     m.LotSizeFilter.MinOrderAmt,
+		MinAmount:       m.LotSizeFilter.MinOrderAmt,
```
Comment (ycdesu) [inline pkg/exchange/bybit/convert.go:41]: These two fields have the same value. Does that expect?

Comment (ycdesu) [reply pkg/exchange/bybit/convert.go:41]: Please also add unit tests for this convertor.

File: pkg/exchange/bybit/convert.go (hunk 1 above)
Comment (bailantaotao) [reply pkg/exchange/bybit/convert.go:32]: sure!

File: pkg/exchange/bybit/convert.go (hunk 2 above)
Comment (bailantaotao) [reply pkg/exchange/bybit/convert.go:41]: This has been confirmed with @c9s . Some exchanges have a `MIN_NOTIONAL` and declare it in the API response. But in BYBIT there is only a minimum amount limit, so we fill him. This value is used to confirm the minimum order amount when trading

Comment (c9s) [reply pkg/exchange/bybit/convert.go:41]: Yes

File: pkg/exchange/bybit/convert.go (hunk 1 above)
Comment (c9s) [reply pkg/exchange/bybit/convert.go:32]: Yeah that would be better

**Review by gx578007 — APPROVED** [review]

**Pull request merged**